Resetting modules to system default. Reseting $MODULEPATH back to system default. All extra directories will be removed from $MODULEPATH.

Lmod is automatically replacing "xl/16.1.1-5" with "gcc/6.4.0".


Due to MODULEPATH changes, the following have been reloaded:
  1) spectrum-mpi/10.3.1.2-20200121

2020-05-14 21:12:25.756589: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.756603: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.756589: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.756589: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.756603: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.756603: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.788547: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.788550: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.788568: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.788568: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.788569: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
2020-05-14 21:12:25.788549: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudart.so.10.1
Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.593620 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.593621 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 21:12:31.593805 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.593620 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 21:12:31.593806 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.593631 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.593631 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.593631 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 21:12:31.593812 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.593806 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.593812 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.593813 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.622897 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.622922 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.622896 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 21:12:31.623091 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.622922 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.622897 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0514 21:12:31.622922 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:117: The name tf.global_variables is deprecated. Please use tf.compat.v1.global_variables instead.

W0514 21:12:31.623110 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.623091 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.623093 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.623111 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.623109 35184372403024 module_wrapper.py:139] From /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/horovod/tensorflow/__init__.py:143: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0514 21:12:31.834953 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.834968 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.834953 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.834968 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.834953 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.835254 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835255 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835255 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835451 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.835266 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835613 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.835266 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835613 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.835798 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.835720 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.836021 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835715 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.835765 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835881 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835956 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835883 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835955 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

W0514 21:12:31.835803 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:17: The name tf.ConfigProto is deprecated. Please use tf.compat.v1.ConfigProto instead.

W0514 21:12:31.836032 35184372403024 module_wrapper.py:139] From /gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod/keras_mnist.py:20: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.

2020-05-14 21:12:31.850954: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:31.850980: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:31.850980: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:31.850982: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:31.851147: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:31.851483: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:31.852400: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x10ba4a1b0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:31.852423: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:31.852487: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x1469cb460 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:31.852519: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:31.852541: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x11d24a960 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:31.852563: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:31.852648: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15102a900 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:31.852670: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:31.852755: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15b1ca920 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:31.852776: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:31.852812: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x13645a2a0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:31.852831: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:31.882010: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:31.882010: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:31.882015: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:31.882337: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:31.882337: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:31.882340: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:32.125640: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:32.125703: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:32.125748: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:32.125703: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:32.126028: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:32.126176: I tensorflow/core/platform/profile_utils/cpu_utils.cc:101] CPU Frequency: 3450000000 Hz
2020-05-14 21:12:32.127157: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x12646a9c0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:32.127180: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:32.127188: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x13decaa20 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:32.127208: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:32.127205: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x12d5aac20 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:32.127232: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:32.127297: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x140b9a780 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:32.127317: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:32.127441: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x14e6caaa0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:32.127460: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:32.127811: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x15c669340 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-05-14 21:12:32.127834: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-05-14 21:12:32.154611: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:32.154609: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:32.154609: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:32.155167: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:32.155173: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:32.155166: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-05-14 21:12:32.540183: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g10n05:31941] *** Process received signal ***
[g10n05:31941] Signal: Aborted (6)
[g10n05:31941] Signal code:  (-6)
[g10n05:31941] [ 0] [0x2000000504d8]
[g10n05:31941] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g10n05:31941] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g10n05:31941] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g10n05:31941] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g10n05:31941] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g10n05:31941] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g10n05:31941] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g10n05:31941] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g10n05:31941] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g10n05:31941] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g10n05:31941] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g10n05:31941] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g10n05:31941] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g10n05:31941] [14] python(_PyCFunction_FastCallDict+0xb0)[0x13cb35890]
[g10n05:31941] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x13cb82754]
[g10n05:31941] [16] python(+0x1c0a2c)[0x13cc20a2c]
[g10n05:31941] [17] python(_PyEval_EvalFrameDefault+0x604)[0x13cc5afc4]
[g10n05:31941] [18] python(PyEval_EvalFrameEx+0x34)[0x13cb13244]
[g10n05:31941] [19] python(+0x1b4f08)[0x13cc14f08]
[g10n05:31941] [20] python(+0x1b6b74)[0x13cc16b74]
[g10n05:31941] [21] python(+0x1c0bac)[0x13cc20bac]
[g10n05:31941] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x13cc5c0dc]
[g10n05:31941] [23] python(PyEval_EvalFrameEx+0x34)[0x13cb13244]
[g10n05:31941] [24] python(+0x1b4f08)[0x13cc14f08]
[g10n05:31941] [25] python(_PyFunction_FastCallDict+0x4dc)[0x13cc177cc]
[g10n05:31941] [26] python(_PyObject_FastCallDict+0x2fc)[0x13cb35efc]
[g10n05:31941] [27] python(_PyObject_Call_Prepend+0x88)[0x13cb3ee48]
[g10n05:31941] [28] python(+0xdef74)[0x13cb3ef74]
[g10n05:31941] [29] python(PyObject_Call+0x74)[0x13cb35664]
[g10n05:31941] *** End of error message ***
2020-05-14 21:12:32.639089: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g10n05:31943] *** Process received signal ***
[g10n05:31943] Signal: Aborted (6)
[g10n05:31943] Signal code:  (-6)
[g10n05:31943] [ 0] [0x2000000504d8]
[g10n05:31943] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g10n05:31943] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g10n05:31943] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g10n05:31943] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g10n05:31943] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g10n05:31943] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g10n05:31943] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g10n05:31943] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g10n05:31943] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g10n05:31943] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g10n05:31943] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g10n05:31943] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g10n05:31943] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g10n05:31943] [14] python(_PyCFunction_FastCallDict+0xb0)[0x12bb35890]
[g10n05:31943] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x12bb82754]
[g10n05:31943] [16] python(+0x1c0a2c)[0x12bc20a2c]
[g10n05:31943] [17] python(_PyEval_EvalFrameDefault+0x604)[0x12bc5afc4]
[g10n05:31943] [18] python(PyEval_EvalFrameEx+0x34)[0x12bb13244]
[g10n05:31943] [19] python(+0x1b4f08)[0x12bc14f08]
[g10n05:31943] [20] python(+0x1b6b74)[0x12bc16b74]
[g10n05:31943] [21] python(+0x1c0bac)[0x12bc20bac]
[g10n05:31943] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x12bc5c0dc]
[g10n05:31943] [23] python(PyEval_EvalFrameEx+0x34)[0x12bb13244]
[g10n05:31943] [24] python(+0x1b4f08)[0x12bc14f08]
[g10n05:31943] [25] python(_PyFunction_FastCallDict+0x4dc)[0x12bc177cc]
[g10n05:31943] [26] python(_PyObject_FastCallDict+0x2fc)[0x12bb35efc]
[g10n05:31943] [27] python(_PyObject_Call_Prepend+0x88)[0x12bb3ee48]
[g10n05:31943] [28] python(+0xdef74)[0x12bb3ef74]
[g10n05:31943] [29] python(PyObject_Call+0x74)[0x12bb35664]
[g10n05:31943] *** End of error message ***
2020-05-14 21:12:32.693457: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g10n05:31939] *** Process received signal ***
[g10n05:31939] Signal: Aborted (6)
[g10n05:31939] Signal code:  (-6)
[g10n05:31939] [ 0] [0x2000000504d8]
[g10n05:31939] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g10n05:31939] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g10n05:31939] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g10n05:31939] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g10n05:31939] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g10n05:31939] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g10n05:31939] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g10n05:31939] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g10n05:31939] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g10n05:31939] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g10n05:31939] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g10n05:31939] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g10n05:31939] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g10n05:31939] [14] python(_PyCFunction_FastCallDict+0xb0)[0x103965890]
[g10n05:31939] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x1039b2754]
[g10n05:31939] [16] python(+0x1c0a2c)[0x103a50a2c]
[g10n05:31939] [17] python(_PyEval_EvalFrameDefault+0x604)[0x103a8afc4]
[g10n05:31939] [18] python(PyEval_EvalFrameEx+0x34)[0x103943244]
[g10n05:31939] [19] python(+0x1b4f08)[0x103a44f08]
[g10n05:31939] [20] python(+0x1b6b74)[0x103a46b74]
[g10n05:31939] [21] python(+0x1c0bac)[0x103a50bac]
[g10n05:31939] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x103a8c0dc]
[g10n05:31939] [23] python(PyEval_EvalFrameEx+0x34)[0x103943244]
[g10n05:31939] [24] python(+0x1b4f08)[0x103a44f08]
[g10n05:31939] [25] python(_PyFunction_FastCallDict+0x4dc)[0x103a477cc]
[g10n05:31939] [26] python(_PyObject_FastCallDict+0x2fc)[0x103965efc]
[g10n05:31939] [27] python(_PyObject_Call_Prepend+0x88)[0x10396ee48]
[g10n05:31939] [28] python(+0xdef74)[0x10396ef74]
[g10n05:31939] [29] python(PyObject_Call+0x74)[0x103965664]
[g10n05:31939] *** End of error message ***
2020-05-14 21:12:32.754137: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g10n05:31942] *** Process received signal ***
[g10n05:31942] Signal: Aborted (6)
[g10n05:31942] Signal code:  (-6)
[g10n05:31942] [ 0] [0x2000000504d8]
[g10n05:31942] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g10n05:31942] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g10n05:31942] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g10n05:31942] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g10n05:31942] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g10n05:31942] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g10n05:31942] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g10n05:31942] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g10n05:31942] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g10n05:31942] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g10n05:31942] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g10n05:31942] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g10n05:31942] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g10n05:31942] [14] python(_PyCFunction_FastCallDict+0xb0)[0x12e725890]
[g10n05:31942] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x12e772754]
[g10n05:31942] [16] python(+0x1c0a2c)[0x12e810a2c]
[g10n05:31942] [17] python(_PyEval_EvalFrameDefault+0x604)[0x12e84afc4]
[g10n05:31942] [18] python(PyEval_EvalFrameEx+0x34)[0x12e703244]
[g10n05:31942] [19] python(+0x1b4f08)[0x12e804f08]
[g10n05:31942] [20] python(+0x1b6b74)[0x12e806b74]
[g10n05:31942] [21] python(+0x1c0bac)[0x12e810bac]
[g10n05:31942] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x12e84c0dc]
[g10n05:31942] [23] python(PyEval_EvalFrameEx+0x34)[0x12e703244]
[g10n05:31942] [24] python(+0x1b4f08)[0x12e804f08]
[g10n05:31942] [25] python(_PyFunction_FastCallDict+0x4dc)[0x12e8077cc]
[g10n05:31942] [26] python(_PyObject_FastCallDict+0x2fc)[0x12e725efc]
[g10n05:31942] [27] python(_PyObject_Call_Prepend+0x88)[0x12e72ee48]
[g10n05:31942] [28] python(+0xdef74)[0x12e72ef74]
[g10n05:31942] [29] python(PyObject_Call+0x74)[0x12e725664]
[g10n05:31942] *** End of error message ***
2020-05-14 21:12:32.811548: F tensorflow/stream_executor/lib/statusor.cc:34] Attempting to fetch value instead of handling error Internal: no supported devices found for platform CUDA
[g10n05:31940] *** Process received signal ***
[g10n05:31940] Signal: Aborted (6)
[g10n05:31940] Signal code:  (-6)
[g10n05:31940] [ 0] [0x2000000504d8]
[g10n05:31940] [ 1] /lib64/libc.so.6(abort+0x2b4)[0x200000182094]
[g10n05:31940] [ 2] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow8internal15LogMessageFatalD1Ev+0x3c)[0x200050e085ac]
[g10n05:31940] [ 3] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN15stream_executor4port17internal_statusor6Helper5CrashERKN10tensorflow6StatusE+0x7c)[0x200050cadd1c]
[g10n05:31940] [ 4] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZNK10tensorflow9XlaDevice6clientEv+0x10c)[0x20004b1baccc]
[g10n05:31940] [ 5] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice22GetDeviceContextLockedEv+0x5c)[0x20004b1bce9c]
[g10n05:31940] [ 6] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow9XlaDevice16UseGpuDeviceInfoEv+0x54)[0x20004b1be244]
[g10n05:31940] [ 7] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow19XlaGpuDeviceFactory13CreateDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x654)[0x20004b1a2be4]
[g10n05:31940] [ 8] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow13DeviceFactory10AddDevicesERKNS_14SessionOptionsERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEPSt6vectorISt10unique_ptrINS_6DeviceESt14default_deleteISE_EESaISH_EE+0x1d4)[0x200073c34c24]
[g10n05:31940] [ 9] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow20DirectSessionFactory10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x314)[0x20004a1d5294]
[g10n05:31940] [10] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/../libtensorflow_framework.so.1(_ZN10tensorflow10NewSessionERKNS_14SessionOptionsEPPNS_7SessionE+0x12c)[0x200073cd376c]
[g10n05:31940] [11] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(TF_NewSession+0x40)[0x200046abd740]
[g10n05:31940] [12] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(_ZN10tensorflow16TF_NewSessionRefEP8TF_GraphPK17TF_SessionOptionsP9TF_Status+0x24)[0x200046221b54]
[g10n05:31940] [13] /sw/summit/ibm-wml-ce/anaconda-base/envs/ibm-wml-ce-1.6.2-3/lib/python3.6/site-packages/tensorflow_core/python/_pywrap_tensorflow_internal.so(+0x2498860)[0x200046198860]
[g10n05:31940] [14] python(_PyCFunction_FastCallDict+0xb0)[0x127195890]
[g10n05:31940] [15] python(_PyCFunction_FastCallKeywords+0x24)[0x1271e2754]
[g10n05:31940] [16] python(+0x1c0a2c)[0x127280a2c]
[g10n05:31940] [17] python(_PyEval_EvalFrameDefault+0x604)[0x1272bafc4]
[g10n05:31940] [18] python(PyEval_EvalFrameEx+0x34)[0x127173244]
[g10n05:31940] [19] python(+0x1b4f08)[0x127274f08]
[g10n05:31940] [20] python(+0x1b6b74)[0x127276b74]
[g10n05:31940] [21] python(+0x1c0bac)[0x127280bac]
[g10n05:31940] [22] python(_PyEval_EvalFrameDefault+0x171c)[0x1272bc0dc]
[g10n05:31940] [23] python(PyEval_EvalFrameEx+0x34)[0x127173244]
[g10n05:31940] [24] python(+0x1b4f08)[0x127274f08]
[g10n05:31940] [25] python(_PyFunction_FastCallDict+0x4dc)[0x1272777cc]
[g10n05:31940] [26] python(_PyObject_FastCallDict+0x2fc)[0x127195efc]
[g10n05:31940] [27] python(_PyObject_Call_Prepend+0x88)[0x12719ee48]
[g10n05:31940] [28] python(+0xdef74)[0x12719ef74]
[g10n05:31940] [29] python(PyObject_Call+0x74)[0x127195664]
[g10n05:31940] *** End of error message ***
ERROR:  One or more process (first noticed rank 9) terminated with signal 6 (core dumped)

------------------------------------------------------------
Sender: LSF System <lsfadmin@batch3>
Subject: Job 97489: <hello> in cluster <summit> Exited

Job <hello> was submitted from host <login3> by user <yngtodd> in cluster <summit> at Thu May 14 21:11:09 2020
Job was executed on host(s) <1*batch3>, in queue <batch>, as user <yngtodd> in cluster <summit> at Thu May 14 21:12:06 2020
                            <42*g10n04>
                            <42*g10n05>
</ccs/home/yngtodd> was used as the home directory.
</gpfs/alpine/proj-shared/med111/yngtodd/src/opt/hello_horovod> was used as the working directory.
Started at Thu May 14 21:12:06 2020
Terminated at Thu May 14 21:12:34 2020
Results reported at Thu May 14 21:12:34 2020

The output (if any) is above this job summary.

